deployment/terraform/assets/cluster.tf

```
@@ -299,13 +307,13 @@ resource "aws_elasticache_cluster" "redis_server" {
   port                 = 6379
   security_group_ids   = [aws_security_group.redis[0].id]
   availability_zone    = var.aws_az
-  subnet_group_name    = var.redis_enabled && length(var.cluster_subnet_ids) > 1 ? aws_elasticache_subnet_group.redis[0].name : ""
+  subnet_group_name    = var.redis_enabled && length(var.cluster_subnet_ids.redis) > 1 ? aws_elasticache_subnet_group.redis[0].name : ""
 }
 
 resource "aws_db_subnet_group" "db" {
   name       = "${var.cluster_name}-db-subnet-group"
   subnet_ids = (length(var.cluster_subnet_ids.database) > 0) ? tolist(var.cluster_subnet_ids.database) : tolist(data.aws_subnets.selected.ids)
```

Subnet groups require 2+ subnets

```suggestion
  subnet_ids = (length(var.cluster_subnet_ids.database) > 1) ? tolist(var.cluster_subnet_ids.database) : tolist(data.aws_subnets.selected.ids)
```